src/utils/schema/allCommonMethods.ts

```diff
@@ -0,0 +1,13 @@
+import { GraphQLError } from 'graphql';
+import i18next from 'i18next';
+
+/**
+ * Check user login or not.
+ * Throw error for user not logged.
+ * @param user login user data
+ */
+export const userNotLogged = (user: any): void => {
```

Comment:
@TaiKamilla 
I don't like that the method is put in a "allCommonMethods" file
+ I don't like the name of the method

it could be
checkUserAuthenticated